data/resources/style.css

```diff
@@ -6,6 +6,11 @@
   font-size: 0.8em;
 }
 
+button.send-message {
+  color: @accent_color;
+  -gtk-icon-size: 20px;
```

Comment:
I think that the problem here is that the new symbolic is gigantic in size. It should be a 16x16 SVG (https://developer.gnome.org/hig/guidelines/ui-icons.html).

I still think that a better fix is tweaking the current icon to our needs. So, rounding the corners and make it fill the whole 16x16 canvas.

Reply:
![image](https://user-images.githubusercontent.com/63008755/195668438-806ada7f-2641-4721-a58e-1e2bb14dbc41.png)

In 16x16 size it looks like this

Reply:
Yeah, that's normal, because the internal icon should have appropriate spacing around the button. To make it appear larger it's more appropriate to use .suggested-action, like it was before.